source/administration/changelog.md

```diff
@@ -4,6 +4,131 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Extended the payload of slash commands to include a map of the users and channels mentioned in the message to their corresponding identifiers.
+ - Added support for recognizing multi-line slash commands without requiring trailing space after the trigger word.
+ 
+#### Administration
+ - Added support for exporting and importing the props of a post.
+
+### Bug Fixes
+ - Fixed an issue where a user's role was not reflected correctly in the Channel Members Modal when it was updated after the modal was opened.
+ - Fixed an issue where a link to Integrations page was not shown if a custom slash command and webhooks were disabled and bot accounts were enabled.
+ - Fixed an issue where verification emails were still sent on servers with SMTP configured when`Enable Email Notifications` and `Require Email Verification` were disabled in the System Console.
+ - Fixed an issue where a user account was still created when inviting a new user to a team with an email address that didn't match the team's allowed domain.
+ - Fixed an issue where System Admins could not access the Teams menu of the System Console.
+ - Fixed an issue where an incorrect time was displayed at midnight when 24-hour clock display was enabled.
+ - Fixed an issue where a channel appeared twice on the channel sidebar if the channels were created with a certain arrangement of characters.
+ - Fixed an issue where pasting a custom theme caused a white screen.
+ - Fixed an issue where a modified Edit Post dialog silently closed on a mouse click outside it.
+ - Fixed an issue where users were unable to drag and drop files on Edge.
+ - Fixed an issue where the autoresponder responded to every bot post.
+ - Fixed data race conditions that occurred when accessing ``Post.Props``.
+ - Fixed an issue where Mattermost was unable to start if a configured mail server was listening but not responding.
+ - Fixed an issue where LDAP sync did not finish if read database replica was enabled.
+ - Fixed a SIGSEGV issue when exporting to CSV.
+ - Fixed an issue where Elasticsearch error was output when running unrelated commands.
+ - Fixed an issue where importing from slack crashed due to invalid memory access or nil pointer dereference.
+ 
+### config.json
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in ``config.json``:
+    - Added ``EnableOpenTracing``, to 
+    - Added ``IdleTimeout``, to 
+    - Added ``ExperimentalChannelSidebarOrganization``, to 
+ - Under ``NotificationLogSettings`` in ``config.json``:
+    - Added ``SMTPServerTimeout``, to 
+ - Added ``Office365Settings`` object to add support for Office365Settings Directory (tenant) Id.
+ - Added ``ExperimentalAuditSettings`` object to
```

Comment:
@wiersgallak @esethna @thefactremains FYI this is the list of config settings that should also be documented at https://docs.mattermost.com/administration/config-settings.html.